packages/input/stories/it-input-time.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta, Story, Canvas, Controls, Description, Subtitle, Title } from '@storybook/addon-docs/blocks';
+import * as Stories from './it-input-time.stories';
+
+<Meta of={Stories} />
+
+# Input Ora
+
+<description>Campo per selezionare un orario all'interno di un form</description>
+
+## Cosa fa
+Il componente input orario, o timepicker, consente all'utente di scegliere un orario, indicando ore e minuti.
+È utile per i flussi di interazione che prevedono la comunicazione di un orario, come la prenotazione di appuntamenti o di posti per un evento. Aiuta a prevenire errori, e rende più facile inserire i dati per l'utente.
+
+[Approfondisci come e quando usare il componente Input Ora](https://designers.italia.it/design-system/componenti/timepicker/)
+
+### Alternative a questo componente
+- [Select](?path=/docs/componenti-form-select--documentazione): se hai bisogno che gli utenti selezionino un orario da una lista di opzioni prestabilite, ad esempio con un intervallo di 5 o 10 minuti.
```

Comment:
Come sopra, finché non è fatto non metterei il link, anche se so che poi son da ripassare. Magari potremmo scriverci tra parentesi che è in lavorazione, mi pare l'avessimo fatto altrove ma non ricordo dove.